.github/workflows/osv-scanner.yml

```diff
@@ -21,15 +21,23 @@ on:
 
 jobs:
   run-osv-scanner:
+
     name: Run OSV Scanner
     runs-on: ubuntu-latest
+
     steps:
+
       - name: Checkout
         uses: actions/checkout@v3
+
+      - name: Pull OSV Scanner Docker image
+        run: docker pull ghcr.io/google/osv-scanner:latest
```

Comment:
You can replace this with `uses: ghcr.io/google/osv-scanner:latest`
Then you don't need to pull the docker image by hand and can include the commands from the next step into this one

Reply:
i did try it in my repo, with this approach bt  i am having trouble generating the `VULNERABILITY.md` file that will contain the results. I implemented as:
```
  - name: Scan for vulnerabilities
    id: osv_scan
    uses: docker://ghcr.io/google/osv-scanner:latest
    with:
      args: --format markdown -r .
    continue-on-error: true

```
 - i cannot place ` grep -vE '^Scanned.*packages$' > VULNERABILITY.md` in args
 - i tried with ${{ steps.osv_scan.outputs.stdout }}, ${{ steps.osv_scan.outputs.result }} but didnot work
 - if we use 3rd party action related to artifacts, it will just make it more complex
 - if we use `outputs:` to define the output of the osv-scanner job then we will need to create another job to use that output

Reply:
Alright, then let us keep the current version